Gems/ROS2/Code/Source/RobotImporter/Pages/PrefabMakerPage.cpp

```diff
@@ -96,8 +98,12 @@ namespace ROS2
         if (!m_spawnPointsInfos.empty())
         {
             int vectorIndex = m_spawnPointsComboBox->currentData().toInt();
+            if (vectorIndex == 0)
+            {
```

Comment:
Add a comment please why this is a bad value

Reply:
Please implement IsDefault method to make it more clear and use constexpr char* DefaultSpawnPointLabel =  "default".